support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -130,42 +130,28 @@ EvolutionSystem:
         SpatialDecayWidth: 17.0152695482514 # From SpEC run: 100.0 / sqrt(34.54)
         Amplitudes: [1.0, 1.0, 1.0]         # From SpEC run: damped harmonic
         Exponents: [2, 2, 2]                # From SpEC run
-    DampingFunctionGamma0:
+    DampingFunctionGamma0: &ConstraintDampingTripleGaussian
       TimeDependentTripleGaussian:
-        Constant: 0.001             # 0.001 / (m_A + m_B)
+        Constant: {{ Gamma0Constant }}
```

Comment:
Note that you _can_ do simple math in these templates, e.g. `Constant: {{ 0.001 / (m_A + m_B) }}`. Maybe that's better than doing it in Python and passing it down? It would certainly make easy expressions a lot less verbose, and probably easier to understand.

Reply:
I saw we already did that for some other parameters in the input file and I didn't like it because it's basically a python variable in a yaml file. And if we have math in the yaml like that it's a bit confusing. I'd prefer to do all the math in the python script and just treat the jinja templates as a find/replace with values. I think that's clearer for users if they ever look at these files (and clearer for me). Only one place to look for math rather than scattered through an input file

Reply:
🤷‍♂️ don't know we can try it this way. I don't think doing easy math in the input file template is confusing, and it would reduce the number of parameters being computed and passed to the template. But I'm fine with doing it either way for now.

Reply:
As someone who in SpEC took every opportunity to do things in input files instead of code, I agree with Kyle: I've learned my lesson, and it's better to discourage users from doing math not in code, because it's easier for them to introduce mistakes, there are often fewer checks and safeguards, and it's more likely to end up with a yaml file not in version control than code